_sass/components/_footer.scss

```diff
@@ -0,0 +1,154 @@
+.c-footer {
+  background-color: $dark-blue;
+  width: 100%;
+
+  &__container {
+    width: 80%;
+    margin: 0 auto;
+    padding: 4rem 0;
+  }
+
+  &__award-container {
+    width: 100%;
+    padding: 4rem 0;
+  }
+
+  &__award-image {
+    width: 5rem;
+    margin: 0 auto;
+    display: block;
+  }
+
+  &__award-text {
+    @include uppercase-white-font;
+    text-align: center;
+    margin: 1.5rem auto 0;
+    max-width: 23.625rem;
+  }
+
+  &__link-lists {
+    @include link-lists-flex;
+  }
+
+  &__link-list {
+    width: 100%;
+    padding: 0 0.625rem 0 0;
+    box-sizing: border-box;
+    margin: 1.25rem 1.25rem 0;
+
+    @media (min-width: $bp-footer-sm) {
+      width: 45%;
+      margin: 1.25rem 0.625rem 0;
+    }
+
+    @media (min-width: $bp-lg) {
+      width: 20%;
+      margin: 1.25rem 0.625rem;
+    }
+
+    @media (min-width: $bp-xl) {
+      width: 14%;
+      margin: 1.25rem 0.625rem 40px;
+    }
+  }
+
+  &__link-title {
+    @include uppercase-white-font;
+    margin: 0 0 1.25rem;
+
+  }
+
+  &__link {
+    color: $medium-blue;
```

Comment:
Could you change this to a bright blue? the contrast of it against the dark blue background doesnt pass the contrast ratio, this hex value, `#0599e3` would pass, which is pretty close to this `$medium-blue`